src/main/java/hudson/tasks/junit/CaseResult.java

```diff
@@ -269,7 +268,7 @@ public static float clampDuration(float d) {
         return Math.min(365.0f * 24 * 60 * 60, Math.max(0.0f, d));
     }
 
-    public static CaseResult parse(SuiteResult parent, final XMLStreamReader reader, String ver)
```

Comment:
As far as I know nothing should be using these new methods from #625, so I made them all package-private, but if that is not true, let me know and I will revert that part of the change.